api/applicant/controllers/applicant.js

```diff
@@ -26,6 +32,7 @@ module.exports = {
       level
     } = ctx.request.body;
 
+    console.log('new application');
```

Comment:
Is this used for debug? Do we need to keep it?

Reply:
Yes, it's used for dedug, could remove it.